### ISSUE TITLE
writeStyle(...) doesn't handle "GEOMTRANSFORM CENTROID" case (again)

### ISSUE THREAD

**Issue by ahermange** [MapServer/MapServer#6219]
## Expected behavior and actual behavior.

when a mapfile is saved the GEOMTRANSFORM "centroid" disapear

![image](https://user-images.githubusercontent.com/6838068/106120476-b663af00-6156-11eb-9936-7d6565333fe7.png)

## Operating system

Windows 10 20H2
Windows Server 2016

## MapServer version and installation method

MapServer version 7.4.3  (windows compilation)
MapServer version 7.7.0-dev (MS4W 4.0.5)

## Solution 

modify this (mapfile.c) 
the value of the parameter 5 is 7 , we have 8 case, the 8 (centroid) will never be tested

 ```
writeKeyword(stream, indent, "GEOMTRANSFORM", style->_geomtransform.type, 7,
                 MS_GEOMTRANSFORM_BBOX, "\"bbox\"",
                 MS_GEOMTRANSFORM_END, "\"end\"",
                 MS_GEOMTRANSFORM_LABELPOINT, "\"labelpnt\"",
                 MS_GEOMTRANSFORM_LABELPOLY, "\"labelpoly\"",
                 MS_GEOMTRANSFORM_LABELCENTER, "\"labelcenter\"",
                 MS_GEOMTRANSFORM_START, "\"start\"",
                 MS_GEOMTRANSFORM_VERTICES, "\"vertices\"",
                 MS_GEOMTRANSFORM_CENTROID, "\"centroid\""
                );
```

to

```
 writeKeyword(stream, indent, "GEOMTRANSFORM", style->_geomtransform.type, 8,
                 MS_GEOMTRANSFORM_BBOX, "\"bbox\"",
                 MS_GEOMTRANSFORM_END, "\"end\"",
                 MS_GEOMTRANSFORM_LABELPOINT, "\"labelpnt\"",
                 MS_GEOMTRANSFORM_LABELPOLY, "\"labelpoly\"",
                 MS_GEOMTRANSFORM_LABELCENTER, "\"labelcenter\"",
                 MS_GEOMTRANSFORM_START, "\"start\"",
                 MS_GEOMTRANSFORM_VERTICES, "\"vertices\"",
                 MS_GEOMTRANSFORM_CENTROID, "\"centroid\""
                );
```

the "labelcenter" is not documented in [https://www.mapserver.org/mapfile/label.html#label-style](url)

**Comment by jmckenna** [comment]
@ahermange can you create a pull request for this?

**Comment by jmckenna** [comment]
(similar to old closed ticket #4480 )

**Comment by jmckenna** [comment]
Documented `labelcenter` through https://github.com/MapServer/MapServer-documentation/commit/849f83ed6820200f2bb98b262e9ed4be86fa2ded Thanks again @ahermange !  Closing.
